python/paddle/distributed/collective.py

```diff
@@ -98,7 +309,7 @@ def _get_global_default_group():
     return _default_group
 
 
-def broadcast(tensor, src, group=0):
+def broadcast(tensor, src, group=0, use_calc_stream=True):
```

Comment:
maybe sync insteads of use_calc_stream which is difficult to understand for users.